m4_sessions/1803-inning/group_3.md

```diff
@@ -1,13 +1,23 @@
 Group 3
 
-Group Members:
+Group Members: Alex Bonder, Jeff Goodall
 
-Name of Session:
+Name of Session: How to be an advocate for those that don't have a voice.
 
 Session Description (Why should students come to this session):
-
+  Advocacy is a driving force in promoting diversity within the workplace.
+  Some people aren't comfortable asking questions even though the environment emphasises that all voices are welcome. How do we overcome this?
+  
+  
 Session Outcomes (What will students learn/leave with?):
+  Leave with the understanding of how someone without a voice might feel day to day. and Some ideas on how to actively participate in being an advocate for those people.
```

Comment:
I like this as an outcome--it is always important to think about the way that people might feel without a voice.